configs/ledger-specific.js

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const {config} = bedrock;
+const {constants} = config;
+const c = bedrock.util.config.main;
+const cc = c.computer();
+
+const cfg = config['ledger-specific'] = {};
+
+cfg.peers = [];
+
+cfg.did = 'did:ledgertest:0e6bdea3-05d2-4d96-a344-df65ed03b55a';
```

Comment:
Do all ledgers need to be associated with a DID?

Reply:
Excellent question, I have never understood why ledgers themselves are identified with DIDs.  @dlongley can you fill us in?

Reply:
There's no reason I can think of as to why a ledger would have to be identified by a DID. In order to root the authority for writing to a ledger in something one would need some kind of ID (assuming we're using zcaps to handling that), but it could be any URI, it doesn't have to be a DID.